src/main/java/percy/command/Parser.java

```diff
@@ -22,6 +22,8 @@ public Command getCommand() {
                 return new DoneCommand(this.getTaskNumber());
             case "list":
```

Comment:
consider enumerations instead of case and switch as I believe enumerations are not case sensitive